iron-input.html

```diff
@@ -130,8 +150,9 @@
     },
 
     /**
-     * Returns true if the value is valid. The validator provided in `validator` will be used first,
+     * Returns true if `value` is valid. The validator provided in `validator` will be used first,
      * then any constraints.
+     * @return {Boolean} True if the value is valid.
```

Comment:
Yeah, I don't know..is it `boolean` and `number` but `Object`? What about `function`?